tests/installation/bootloader_s390.pm

```diff
@@ -120,7 +120,7 @@ sub prepare_parmfile {
         set_var('AUTOYAST', $url);
     }
     if (get_var('AGAMA_AUTO')) {
-        my $url = data_url(get_var('AGAMA_AUTO'));
+        my $url = shorten_url(data_url(get_var('AGAMA_AUTO')));
```

Comment:
this function cannot be used in O3, needs to be conditional for now.

Reply:
yes. will change it.